cns/logger/cnslogger_windows.go

```diff
@@ -0,0 +1,34 @@
+package logger
+
+import (
+	"github.com/Azure/azure-container-networking/zapetw"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+const (
+	// LogPath is the path where log files are stored.
+	LogPath         = ""
+	etwCNSEventName = "AzureCNS"
+)
+
+func GetPlatformCores(loggingLevel zapcore.Level) (zapcore.Core, error) {
+	etwcore, err := GetETWCore(loggingLevel)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get ETW core")
+	}
+	return etwcore, nil
+}
+
+func GetETWCore(loggingLevel zapcore.Level) (zapcore.Core, error) {
+	encoderConfig := zap.NewProductionEncoderConfig()
+	encoderConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+	jsonEncoder := zapcore.NewJSONEncoder(encoderConfig)
```

Comment:
I think we've discussed this elsewhere, but JSON shouldn't be assumed. The whole power of structured logging is that it's structured--not structured _in a particular way_. The particulars can be configured by the user (maybe they want XML).